tests/test_detect_repo_type.py

```
@@ -499,6 +456,102 @@ def test_services_dir_scores_monorepo(self, tmp_path):
+
+    _script = (
+        pathlib.Path(__file__).resolve().parent.parent
+        / "skills"
+        / "repo-indexer"
+        / "scripts"
+        / "detect-repo-type.py"
+    )
+
+    def test_output_type_prefix(self, tmp_path):
+        """First line: 'TYPE: <word> (confidence: <float>)'."""
+        result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)